internal/pkg/term/selector/selector.go

```diff
@@ -89,138 +89,134 @@ var presetSchedules = []prompt.Option{
 	{Value: yearly, Hint: "At midnight, Jan 1st UTC"},
 }
 
-// Prompter wraps the methods to ask for inputs from the terminal.
-type Prompter interface {
+// prompter wraps the methods to ask for inputs from the terminal.
+type prompter interface {
 	Get(message, help string, validator prompt.ValidatorFunc, promptOpts ...prompt.PromptConfig) (string, error)
 	SelectOne(message, help string, options []string, promptOpts ...prompt.PromptConfig) (string, error)
 	SelectOption(message, help string, opts []prompt.Option, promptCfgs ...prompt.PromptConfig) (value string, err error)
 	MultiSelect(message, help string, options []string, validator prompt.ValidatorFunc, promptOpts ...prompt.PromptConfig) ([]string, error)
 	Confirm(message, help string, promptOpts ...prompt.PromptConfig) (bool, error)
 }
 
-// AppEnvLister wraps methods to list apps and envs in config store.
-type AppEnvLister interface {
+type appEnvLister interface {
 	ListEnvironments(appName string) ([]*config.Environment, error)
 	ListApplications() ([]*config.Application, error)
 }
 
-// ConfigWorkloadLister wraps the method to list workloads in config store.
-type ConfigWorkloadLister interface {
+type configWorkloadLister interface {
 	ListServices(appName string) ([]*config.Workload, error)
 	ListJobs(appName string) ([]*config.Workload, error)
 	ListWorkloads(appName string) ([]*config.Workload, error)
 }
 
-// ConfigLister wraps config store listing methods.
-type ConfigLister interface {
-	AppEnvLister
-	ConfigWorkloadLister
+type configLister interface {
+	appEnvLister
+	configWorkloadLister
 }
 
-// WsWorkloadLister wraps the method to get workloads in current workspace.
-type WsWorkloadLister interface {
+type wsWorkloadLister interface {
 	ListServices() ([]string, error)
 	ListJobs() ([]string, error)
 	ListWorkloads() ([]string, error)
 }
 
-// WsPipelinesLister is a pipeline lister.
```

Comment:
Why do we remove comments for some interfaces but not all of them?

Reply:
Some interfaces's purpose I think is a little ambiguous, whereas others are clear judging from the signature. I kept the comments for the first group of interfaces.